templates/config.json

```diff
@@ -137,6 +137,10 @@
       "id": "program",
       "file": "policies/program.md",
       "name": "Security Program Overview",
+      "instructionsSummary": "Confirm/tailor link to Employee Handbook.",
```

Comment:
Minor: `Confirm/update` or `Confirm/customize` reads better to me than `Confirm/tailor`

Reply:
I am rewording every Confirm/update to be a question to the customer that they can confirm/update.